test/run

```diff
@@ -19,9 +19,9 @@ test_mypy() {
 }
 
 test_ruff
-test_mypy
+#test_mypy
```

Comment:
Is this for testing?  (and x2 more below)

Reply:
Ouch, yes.